atd-etl/cris_import/.gitignore

```diff
@@ -1 +1,2 @@
 env
+*sqlite
```

Comment:
I find that I end up with these sqlite files floating around from inspecting them off S3. This prevents them from ever getting into `git`, but if they do, it's not a huge deal. No secrets in there.